ScoutSuite/frontend/src/components/Partial/PartialValue/style.scss

```diff
@@ -7,6 +7,9 @@
 
   .issue {
     font-weight: 700;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
```

Comment:
@SamuelMeilleur there is probably a better way to do this but it work fine for now